crates/goose-cli/src/commands/configure.rs

```diff
@@ -112,11 +112,13 @@ pub async fn handle_configure(
         estimate_factor: None,
     };
 
+    // Set environment variables for provider configuration
+    set_provider_env_vars(&provider_name, &profile);
```

Comment:
this is a patch for now to keep things working as is, but in the future will rework this part to a more native solution